C05 kks dialogue (#103)
IllusionMods/KKS-Translation: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c05/00/55/translation.txt

//
// Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
//

> 「昨日ね、かれらから『コードネームで呼び合ってみろ』ってメッセージが届いたの。だからあなたのコードネームを考えてみたんだ」=“They sent me a message yesterday. It said, "call each other by your code names".”

Reviewer: ```suggestion
「昨日ね、かれらから『コードネームで呼び合ってみろ』ってメッセージが届いたの。だからあなたのコードネームを考えてみたんだ」=“‘They’ sent me a message yesterday. It said, "call each other by your code names". And so, I've thought of a code name for you..”
```

> 「えらく唐突だね……。ちなみにコードネームってどんなの？ 　試しに呼んでみてくれないかな？」=“That's... abrupt. By the way, what's your code name?”

Reviewer: ```suggestion
「えらく唐突だね……。ちなみにコードネームってどんなの？ 　試しに呼んでみてくれないかな？」=“That's... abrupt. So what's the code name by the way?”
```

「うん、いいよ。……[Pあだ名]」=“Yeah, okay... [Pあだ名].”

> （……思ってたより普通だ。普通の愛称だ。もっと突拍子のないものを付けられるかと）=(...that's more normal than I expected. Quite an ordinary nickname. Maybe I should add something outlandish.)

Reviewer: ```suggestion
（……思ってたより普通だ。普通の愛称だ。もっと突拍子のないものを付けられるかと）=(...that's more normal than I expected. Quite an ordinary nickname. I thought it'd be more outlandish.)
```

「……何かリアクションしてほしい」=“...I wanted more of a reaction.”
「あっ、ごめんね。すごくいいと思うよ、ありがとう、[Hあだ名]」=“Ah, sorry. I think that's great, thank you, [Hあだ名].”
「……？　もしかして、それがわたしのコードネーム？」=“...? Is that, perhaps, my codename?”

> 「あ、うん。『呼び合ってみろ』っていうのがメッセージだったんでしょ？　だから俺も考えてみたんだけど、どうかな？」=“Ah, yeah. The message said "call each other," didn't it? So, I gave it a thought... what do you think?”

Reviewer: ```suggestion
「あ、うん。『呼び合ってみろ』っていうのがメッセージだったんでしょ？　だから俺も考えてみたんだけど、どうかな？」=“Ah, yeah. The message said "call each other," didn't it? So, I thought of one too... what do you think?”
```
